vulnfeeds/cpp/main.go

```diff
@@ -87,6 +87,8 @@ var VendorProductDenyList = []VendorProduct{
 	{"netapp", "ontap_select_deploy_administration_utility"},
```

Comment:
I almost wonder if we should completely blocklist anything "netapp"

Reply:
Done so in 2d5d59d